ansible/group_vars/proxy-hosts.yml

```diff
@@ -293,9 +293,16 @@ _nginx_proxy_sites:
     nginx_proxy_additional_directives:
     - "add_header Access-Control-Allow-Origin $allow_origin"
     # Study redirects
-    - "if ($request_uri ~ /search/\\?query=Name:idr0068) {
+    - "if ($request_uri ~ /search/\\?query=Name:(.*)68) {
```

Comment:
At some point we'll end up with `idr0168` and this will break, can the regex check for something like `0*`?

Reply:
The trouble is that URLs of type https://idr.openmicroscopy.org/search/?query=Name:68 are now being used in the wild and we need these to redirect until the data is published.

I have raised your exact issue with the increasing accession numbers. For now I'll deploy but happy to have this discussed on weekly call.

Reply:
e14aef4 should now constraint the regex to only match the two known variants of the search URLs i.e.
 https://idr-next.openmicroscopy.org/search/?query=Name:idr0068 and https://idr-next.openmicroscopy.org/search/?query=Name:68. As suggested, https://idr-next.openmicroscopy.org/search/?query=Name:168 should now return no found study as expected

Reply:
Instead of using the wildcard if statement, which could cause a problem with 168 etc, can we take search term and remove any non-numeric characters then convert to an integer which would make 68 unique?

Reply:
@francesw can you retest the latest version deployed on `idr-next` which should only redirect `Name:68` and `Name:idr0068` ?

Reply:
Tested, works as expected.

Reply:
@francesw I think a good longer term fix is to handle this as a new OMERO.web app where we've got the full manipulative of Python, instead of doing this in Nginx.